SwiftLint.xcodeproj/xcshareddata/xcschemes/swiftlint.xcscheme

```diff
@@ -38,6 +38,17 @@
                BlueprintName = "SwiftLintFrameworkTests"
                ReferencedContainer = "container:SwiftLint.xcodeproj">
             </BuildableReference>
+            <SkippedTests>
+               <Test
+                  Identifier = "AttributesRuleTests/testSwift2Attributes()">
```

Comment:
they did but I've refactored them 😅 
why do I still think Xcode would do the right thing? 😬